apps/prairielearn/src/ee/pages/instructorInstanceAdminBilling/instructorInstanceAdminBilling.test.ts

```diff
@@ -60,9 +60,9 @@ describe('instructorInstanceAdminBilling', () => {
 
   it('forbids disabling student billing if enrollment limit exceeded', async () => {
     await features.runWithGlobalOverrides({ 'course-instance-billing': true }, async () => {
+      await enrollRandomUsers('1', 10);
```

Comment:
This was a fun one! So, obviously enrolling 10 users after setting the enrollment limit to 1 should fail, which it did. But why was the test passing if it should only pass if the enrollment limit was actually exceeded? Because the instructor was previously automatically enrolled in the course upon visiting the billing settings page 🙃

Just goes to show that enrollment limit enforcement is definitely not working!